README.md

```diff
@@ -33,15 +33,23 @@ Install the plugin with `yarn add --dev serverless-dynalite-local`. Add
 `serverless-dynalite-local` to your `serverless.yml` `plugins` key, BEFORE
 `serverless-offline`.
 
-The server will start up at `http://localhost:4567` and use `dynalite/` as a
-data directory. Connect to the database like this:
+By default the server will start up at `http://localhost:4567` and use
+`dynalite/` as a data directory. Connect to the database like this:
 
 ```typescript
 const client = new DynamoDBClient({
   endpoint: "http://localhost:4567",
 });
 ```
 
+The following options are supported in `custom.dynalite` in `serverless.yml`:
+
+| Option | Description | Type | Default value |
```

Comment:
Thank you!! I knew this was going to be useful one day but I decided to skip it for the first version.